dev-docs/modules/floors.md

```diff
@@ -265,31 +265,204 @@ pbjs.setConfig({
-| data.values."floor" | float | the floor value for this key | - |
-| data.default | float | Floor used if no matching rules are found | - |
+| data.schema.delimiter | string | Character separating the floor keys. | '\|' |
+| data.schema.fields | array of strings | Supported values are: gptSlot, adUnitCode, mediaType, size | - |
+| data.values | key / values | A series of attributes representing a hash of floor data in a format defined by the schema object. | - |
+| data.values."rule key" | string | Delimited field of attribute values that define a floor. | - |
+| data.values."rule floor value" | float | The floor value for this key. | - |
+| data.default | float | Floor used if no matching rules are found. | - |
+
+
+
+### Schema 2
+
+Schema 2 allows floors providers to A / B one or more floor groups, determined at auction time.
+
+The following principles apply to schema 2:
+- The below attributes are required:
+    - data.floorsSchemaVersion to be set to 2
+    - A valid modelGroups object must be set
+    - The field modelGroups.modelWeight is required for each model group
+        - If one of the model weights is missing, no schema 2 floor will be set and the Floors Module will look in other locations for floor definitions
+- If common attributes are set in both the modelGroups and root level of the data object, modelGroups attributes prevail
+- The Schema 2 data model can only be applied in Package level (i.e. directly in setConfig) or Dynamic level
+- Sampling weights are applied at the auction level. Each new auction the dice will be rolled
+- If the data.modelGroups object and the data.values (schema 1 field) are set, the data.floorsSchemaVersion will dictate what schema version is applied
+
+
+While some attributes are common in both schema versions, for completeness, all valid schema 2 attributes are provided:
+
+{: .table .table-bordered .table-striped }
+| Param | Type | Description | Default |
+|---+---+---+---+---|
+| enforcement | object | Controls the enforcement behavior within the Floors Module.| - |
+| enforcement.enforceJS | boolean | If set to true, the floors module will provide floors to bid adapters for bid request matched rules and suppress any bids not exceeding a matching floor. If set to false, the prebid floors module will still provide floors for bid adapters, but there will be no floor enforcement.| true |
+| enforcement.enforcePBS | boolean | If set to true, the Prebid.js floors module will signal to Prebid Server to pass floors to it’s bid adapters and enforce floors. If set to false, Prebid.js should still pass matched bid request floor data to Prebid Server, however no enforcement will take place. | false |
+| enforcement.floorDeals | boolean | Enforce floors for deal bid requests. | false |
+| enforcement.bidAdjustment | boolean | Adjust floors passed to Bid Aapters. If bid adjustment is passed to Prebid Server and flag set is not set to false | true |
```

Comment:
Same as above, incomplete sentence here.

Reply:
Read the above. I appreciate you stepping up to apply a second set of eyes and clear up ambiguous language as well as provide wordsmith.